package.json

```diff
@@ -25,6 +25,7 @@
     "proptypes": "cross-env BABEL_ENV=development babel-node -i \"/node_modules/(?!@mui)/\" -x .ts,.tsx,.js ./docs/scripts/generateProptypes.ts",
     "size:snapshot": "node --max-old-space-size=2048 ./scripts/sizeSnapshot/create",
     "size:why": "yarn size:snapshot --analyze --accurateBundles",
+    "t": "node test/cli.js",
```

Comment:
Since we are adding more components, and the data grid seems to be "one of its kind" by requiring a layout for many of its tests, it seems relevant to add the `yarn t DateField` helper to have a better test DX.